canonical/ubuntu-advantage-client
# tests: organize behave steps

## Note
This is only intended to organize our existing steps, not to change any of them.
After they're organized (after this is merged), we will be able to consolidate or remove unneeded steps or do other refactors more easily

## Test Steps

Integration tests should still pass.
Check my organization and tell me if it doesn't make sense or if something is mis-categorized or anything like that.

## Desired commit type


## Checklist
